docker-compose.yml

```diff
@@ -22,8 +21,8 @@ services:
       - DB_PASS=${DB_PASS}
       - DB_USER=${DB_USER}
       - LOGGING_LEVEL=${LOGGING_LEVEL}
-      - WZDX_ENDPOINT=${WZDX_ENDPOINT}
-      - WZDX_API_KEY=${WZDX_API_KEY}
+      - CDOT_FEED_ENDPOINT=${CDOT_FEED_ENDPOINT}
+      - CDOT_FEED_API_KEY=${CDOT_FEED_API_KEY}
```

Comment:
(non-blocking) Is this just a change to the name of the environment variables, or is there a new endpoint we're pointing to?

Reply:
This is just a change to the name of the environment variables. Each translator uses the same base endpoint and API key to access their data feed. I've updated these names to be more generic to reflect this.